shaders/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ endif()
 # ==================================================================================================
 set(SHADERS
         src/ambient_occlusion.fs
+        src/bloom.fs
```

Comment:
This shouldn't be added here, since `bloom.fs` isn't part of `matc`. See the comment below on line 51. You should add `bloom.fs` to this list.